cluster/common.sh

```diff
@@ -99,7 +109,10 @@ function create-kubeconfig() {
     "${kubectl}" config set-credentials "${CONTEXT}" "${user_args[@]}"
   fi
   "${kubectl}" config set-context "${CONTEXT}" --cluster="${CONTEXT}" --user="${CONTEXT}"
-  "${kubectl}" config use-context "${CONTEXT}"  --cluster="${CONTEXT}"
+
+  if [[ "${SECONDARY_KUBECONFIG}" != "" ]];then
```

Comment:
Sorry maybe I am not understanding this correctly, if SECONDARY_KUBECONFIG = true, the following command of setting the context should not be run, right?
Thats not what this condition will do